tests/test_cuda_targeted_insertion_mover.py

```diff
@@ -501,8 +501,8 @@ def test_targeted_insertion_brd4_rbfe_with_context(
     box = initial_state.box0
 
     bps = initial_state.potentials
-    summed_pot = get_bound_potential_by_type(initial_state.potentials, SummedPotential)
```

Comment:
Not for this line: Having tried this change on my own, you will need to unpack the potentials stored in the pickle object used in `test_targeted_insertion_invalid_sample_bug` in this file.

Reply:
Updated the pkl using the code shown in https://github.com/proteneer/timemachine/pull/1384/files#diff-1fe4d63075a510de47b049019ef7176bce26001f8bc9b25e1afaaa5c99b9ba49R1080 .